docs/topics/dokka-migration.md

```diff
@@ -438,8 +438,75 @@ DGP v2 now supports Gradle build cache and configuration cache, improving build
 * To enable build cache, follow instructions in the [Gradle build cache documentation](https://docs.gradle.org/current/userguide/build_cache.html#sec:build_cache_enable).
 * To enable configuration cache, follow instructions in the [Gradle configuration cache documentation](https://docs.gradle.org/current/userguide/configuration_cache.html#config_cache:usage:enable ).
 
+## Troubleshoot your migration
```

Comment:
Hmm 🤔 - we could have Dokka migration troubleshooting instead? - or Troubleshooting Dokka migration? 

Alternatively, Dokka migration troubleshooting tips?

Reply:
I'll leave it as just "Troubleshooting", what do you think?